discord/gateway_intents.go

```diff
@@ -42,6 +42,9 @@ const (
 	GatewayIntentsAll = GatewayIntentsNonPrivileged |
 		GatewayIntentsPrivileged
 
+	GatewayIntentsDefault = GatewayIntentGuilds |
```

Comment:
I don't think it's good practice to provide defaults for Intents, it's something that should be explicitly set by the user.

Reply:
then default would be `0`
idk if thats better

Reply:
I disagree, imo it should throw an error if not provided

Reply:
that would require it to be a pointer which is ugly >.>

Reply:
then defaulting to 0 is better

Reply:
oki